### ISSUE TITLE
Installing heatshrink fails

### ISSUE THREAD

**Issue by kalouantonis** [johan-sports/pyheatshrink#11]
Installing heatshrink on a clean machine fails if Cython is not already installed:

```
Downloading/unpacking heatshrink
  Downloading Heatshrink-0.2.3.tar.gz (57kB): 57kB downloaded
  Running setup.py (path:/vagrant_data/johan_sports_env/build/heatshrink/setup.py) egg_info for package heatshrink
    Traceback (most recent call last):
      File "<string>", line 17, in <module>
      File "/vagrant_data/johan_sports_env/build/heatshrink/setup.py", line 2, in <module>
        from Cython.Build import cythonize
    ImportError: No module named Cython.Build
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 17, in <module>

  File "/vagrant_data/johan_sports_env/build/heatshrink/setup.py", line 2, in <module>

    from Cython.Build import cythonize

ImportError: No module named Cython.Build
```

**Comment by kalouantonis** [comment]
As a quick fix, install Cython before running the install for heatshrink

**Comment by kalouantonis** [comment]
Fixed by PR #12
